06-Tables/Advanced_Tables.ipynb

```diff
@@ -154,18 +150,22 @@
    },
    "outputs": [],
    "source": [
-    "sources = photutils.datasets.make_random_gaussians(\n",
-    "    flux_range=(1000, 10000), n_sources=5, \n",
-    "    xmean_range=(10, 190), ymean_range=(10, 190),\n",
-    "    xstddev_range=(2, 5), ystddev_range=(2,5),\n",
-    "    random_state=1)"
+    "from collections import OrderedDict\n",
```

Comment:
Does this need to be an `OrderedDict` object?

Using `dict` will do almost the same thing:
```
param_ranges = dict([('flux', [1000, 10000]), ('x_mean', [10, 190]), ... ])
```

Reply:
Sorry about that, I was just basing this off a code example, I confirmed it works with dict(), so better to use that and avoid the unnecessary import of `OrderedDict`.